packages/playground-examples/copy/ja/3-7/Types and Code Flow/Recursive Type References.ts

```diff
@@ -0,0 +1,39 @@
+//// { compiler: {  }, order: 2 }
+
+// タイプを使うかインターフェースを使うかはそれぞれの機能の制約に依るところがあります。
+// 3.7では、タイプに関する制約のうち、インターフェースにはない制約が削除
```

Comment:
IMO
type aliasを「タイプ」にしてしまうとなんのことか分からなくなってしまいそうだったので
```suggestion
// 3.7では、type aliasに関する制約のうち、interfaceにはない制約が削除
```